### DISCUSSION TITLE
508 Compliance - Search Field Tabbing Not in Order

### DISCUSSION THREAD

**Discussion by bsherman101** [twbs/bootstrap#36708] [Q&A]
Hello,

I am using bootstrap-table extensively and my application needs to be 508 compliant.

The search text field is displayed to the left of the button toolbar but in the html it is rendered afterwards.

This causes the search field to be skipped over when tabbing and it first cycles through all the buttons in the toolbar and then jumps back to the search field.

Is there an easy way to either make the search field display to the right of the toolbar or render the search field first so it tab's first?

Thanks for the help.

**Comment by florianlacreuse** [comment]
Is "bootstrap-table" a third-party component? If so, you should ask them directly.

**Reply by bsherman101** [reply]
Yes, sorry.  I mistakenly thought this was the bootstrap-table discussion forum.  I'll repost there.
